packages/sdk/src/marketplaces/Opensea.ts

```diff
@@ -121,28 +122,28 @@ export class Opensea implements Marketplace<Order> {
     return createOrder(params);
   }
 
-  /* eslint-disable camelcase */
   async getOrders({
     makerAsset,
     maker,
     takerAsset,
     taker,
   }: GetOrdersParams = {}): Promise<Order[]> {
-    const query: OrderQuery = {
+    const query: Omit<OrdersQueryOptions, "limit"> = {
+      // https://github.com/ProjectOpenSea/opensea-js/blob/master/src/api.ts#L106 limit is currently omitted here
+      protocol: "seaport",
       maker,
-      sale_kind: SaleKind.FixedPrice,
       taker,
+      side: ORDER_SIDE_SELL,
     };
 
     if (!makerAsset && !takerAsset) {
       // Query in sequence instead of Promise.all to avoid getting rate limited.
       const buyOrdersResp = await this.seaport.api.getOrders({
         ...query,
-        side: OrderSide.Buy,
+        side: ORDER_SIDE_BUY,
```

Comment:
V4 killed `OrderSide` enum, now it's a `'ask' | 'bid'` type, so I created 2 const below.